docs.bruno/Testing.bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: Testing
+  type: http
+  seq: 1
+}
+
+get {
```

Comment:
Consider adding more details to the test, such as expected response status and body. This will make the test more robust and meaningful. [medium]